### ISSUE TITLE
Naming Inconsistency

### ISSUE THREAD

**Issue by strarsis** [silverstripe/silverstripe-framework#10596]
## Affected Version
Silverstripe v4.x.

## Description
[According to 4.x changelog/migration notes](https://docs.silverstripe.org/en/4/changelogs/4.0.0/#:~:text=Renamed%20%24combine_files%20to%20%24combinedFiles) the `Requirements::combine_files()` method was renamed to `Requirements::combineFiles()`.
However, the [v4.x SilverStripe guide](https://docs.silverstripe.org/en/4/developer_guides/templates/requirements/#combining-files) still uses the method name `Requirements::combine_files()` instead of the new `Requirements::combineFiles()`.
This is also the case for the v4.x API reference, where some parts still use this old `combine_files()` method name:
https://api.silverstripe.org/4/SilverStripe/View/Requirements_Backend.html#:~:text=Example%20for%20combined%20JavaScript
https://api.silverstripe.org/4/SilverStripe/View/Requirements_Backend.html#:~:text=Example%20for%20combined%20CSS

The source search finds occurrences of `combine_files` (from which also the docs are generated):
https://github.com/silverstripe/silverstripe-framework/search?q=combine_files

**Comment by lozcalver** [comment]
I think the confusion here is between two different classes which name the methods differently - `Requirements::combine_files()` and `Requirements_Backend::combineFiles()`. One just calls the other:

https://github.com/silverstripe/silverstripe-framework/blob/dcdc25500be729f4340d42f8a4fc797461f862f3/src/View/Requirements.php#L392-L398

I think all the docs you’ve linked are actually correct - the 4.x changelog says that `Requirements_Backend` has the renamed methods (not `Requirements`), the guides/API docs using `Requirements::combine_files()` are correct too.

I suppose we could look at making this consistent for 5.x?

**Comment by strarsis** [comment]
@kinglozzer: Yes, this causes confusion! Making the naming conventions and those names the same among different classes surely would help!
